Even though we build for a specific Python version, poetry will error out if it
can't resolve dependencies for all supported Python versions.  This bumps the
minimum version to the version we currently use, to reduce the set of versions
that libraries need to match (specifically to upgrade cryptography in this
instance).
---
diff --git a/pyproject.toml b/pyproject.toml
index e7097fa..4c69c3b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -46,7 +46,7 @@ medusa = { reference = "medusa.medusacli:cli", type = "console" }
 medusa-wrapper = { reference = "medusa.scripts.medusa_wrapper:main", type = "console" }
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.11"
 python-dateutil = "2.8.1"
 click = "8.1.7"
 click-aliases = "1.0.1"
